src/eos-metrics-instrumentation.c

```diff
@@ -351,6 +338,22 @@ record_stop_for_login (GQuark   session_id_quark,
                                           NULL /* auxiliary_payload */);
 }
 
+static void
+add_session (const char *session_id,
+             const char *session_path,
+             guint32 user_id)
+{
+  if (!add_session_to_set (session_id))
```

Comment:
nit: Recommend matching the existing indentation in this file even though it violates the style guide. At some point I will probably fix it all at once.